src/test/rustdoc/issue-19190.rs

```diff
@@ -23,9 +23,6 @@ impl Deref for Bar {
     fn deref(&self) -> &Foo { loop {} }
 }
 
-// @has issue_19190/Bar.t.html
 // @has issue_19190/struct.Bar.html
-// @has - '//*[@id="foo.v"]' 'fn foo(&self)'
```

Comment:
This PR doesn't delete the legacy *HTML IDs*, does it? Why did you remove it from all these tests?

Reply:
I went a bit too strongly on the tests haha.